task/buildah-remote/0.2/buildah-remote.yaml

```diff
@@ -0,0 +1,519 @@
+apiVersion: tekton.dev/v1
```

Comment:
In order for these to pass CI, we will need to update the verification process

https://github.com/redhat-appstudio/multi-platform-controller/blob/e9bab24a0a21c6e0a10b285dd7df385d178c1d61/cmd/taskgen/main.go
https://github.com/redhat-appstudio/build-definitions/blob/main/hack/generate-buildah-remote.sh

Reply:
This needs some significant updates, I will have a look. 

Note that I don't think this will affect CI yet, as the verification won't be checking this new task yet.